### PR TITLE
Added Basic WMS

### PR DESCRIPTION
**Pull request by m-mohr**
Partially implements #8, I think.

### CODE REVIEW

File: examples/item.json [hunk 1]
```diff
@@ -57,6 +57,16 @@
...
     },
+    {
+      "href": "https://maps.example.com/wms",
+      "rel": "wms",
+      "type": "text/xml",
+      "title": "RGB composite visualized through a WMS",
+      "wms:layers": [
+        "rgb"
+      ],
+      "wms:styles": []
```
Comment (santilland) [inline examples/item.json]: if the styles parameter is optional it would be probably best to completely leave it out if its empty?

Comment (m-mohr) [inline examples/item.json]:
```suggestion
```

Comment (m-mohr) [reply examples/item.json]: Doesn't really make a difference, I think, but I removed it.

**Review by santilland — APPROVED** [review]
Nice! Looks good to me!

File: json-schema/schema.json [hunk 2]
```diff
@@ -95,6 +101,44 @@
...
+                  "items": {
+                    "type": "string",
+                    "minLength": 1
+                  }
+                },
+                "wms:styles": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "minLength": 1
```
Comment (constantinius) [inline json-schema/schema.json:131]: In WMS, `styles` can be left empty, to use the default style for each layer.

Comment (m-mohr) [reply json-schema/schema.json:131]: Thus you can provide an empty array.

Comment (constantinius) [reply json-schema/schema.json:131]: Oh, I thought `minLength` means that at least one element has to be provided.

Comment (m-mohr) [reply json-schema/schema.json:131]: No, that's the min length of the string.

**Pull request merged**